build-tools/packages/build-cli/package.json

```diff
@@ -121,6 +121,7 @@
 		"@types/inquirer": "^8.2.6",
 		"@types/mocha": "^9.1.1",
 		"@types/node": "^14.18.53",
+		"@types/prettier": "^2.7.3",
```

Comment:
What does this help with? Other packages don't seem to use it.

Reply:
noImplicitAny requires it (I was wrong about no changes being needed). prettier is imported and called programmatically in the generate:changeset command.